src/rules/common-rules.ts

```diff
@@ -0,0 +1,9 @@
+// Note: these date rules are a little different from other rule files and do not
+// return an array of rules.  This was done to allow more flexibility in mixing
+// and matching what date rules to use.
+export const CommonRules = {
```

Comment:
I like it.

Just a note that, if it's not an array then it's singular not plural.

Reply:
Oh nvm again. This is an object of rules. All good. Sigh.